apps/nextjs/README.md

```diff
@@ -0,0 +1,113 @@
+# plone-nextjs
+
+This is a PoC of a NextJS 13 app using the app router and the upcoming `@plone/client` and `@plone/components`. This is intended to serve as playground for the development of both packages.
```

Comment:
Since you use NextJS 14 already I think that NextJS app would be enough without specifying the version number making it easier to upgrade at any time

Reply:
@ichim-david I just improved the README